scripts/change_license.py

```diff
@@ -0,0 +1,44 @@
+"""
```

Comment:
Feels like we'll never need this script again?  Why commit it?

Reply:
To run on pending PRs?

Reply:
It might be useful to run something similar in the CI that checks (or fixes) whether a correct license is present in the files being committed.

Reply:
Yeah, I was meant to keep it for any pending PRs as I saw one of them is adding new file. but I feel now that we could only mention that change on any current PR adding new files and it should be simple change. so, I think I will take that approach.

Reply:
> It might be useful to run something similar in the CI that checks (or fixes) whether a correct license is present in the files being committed.

Yeah, good point, I think we could add a check to CI to check for license using the regex in the script.